mautic/mautic
# Code mode slot - placeholder

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6442
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Code Mode block without content, or with content without input (scripts, css) are not showen in builder, then cannot select it for editation. This PR added gray block with small icon in right corner for code mode block.

You can test with https://mautibox.com/6454

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create an email or LP
2. Choose a theme (not the code mode)
3. Drag and drop the code mode bloc
4. Insert a script inside
5. Save the email or LP
6. Go back on the builder and see that you won't be able to the code mode bloc anymore in the builder.
 

#### Steps to test this PR:
1.  Apply PR and `run php app/console m:a:g`. Mautibox is in developement mode, not require this step.
2.  Repeat all steps and see gray placeholder in CodeMode slot area

![image](https://user-images.githubusercontent.com/462477/44022597-5f956930-9ee8-11e8-9bd8-d8a5aef0e7d5.png)
